Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -141,20 +190,70 @@ final class PledgePaymentPlanOptionView: UIView {
   @objc private func onOptionTapped() {
     self.viewModel.inputs.optionTapped()
   }
+
+  @objc private func onTermsOfUseButtonTapped() {
+    self.viewModel.inputs.termsOfUseTapped()
+  }
+
+  // MARK: - Functions
+
+  private func setupIncrementsStackView(with increments: [PledgePaymentIncrementFormatted]) {
+    var dateLabels: [UILabel] = []
+    increments.forEach { increment in
+      let incrementStackView = UIStackView()
+      applyIncrementStackViewStyle(incrementStackView)
+
+      let titleLabel = UILabel()
+      applyIncrementTitleLabelStyle(titleLabel)
+      titleLabel.text = increment.title
```

Comment:
nit: Can we rename this to something like `chargeNumberLabel` or something that makes it a bit clearer?